firebase-perf/src/main/java/com/google/firebase/perf/network/InstrHttpInputStream.java

```diff
@@ -183,11 +180,13 @@ public long skip(final long byteCount) throws IOException {
       if (timeToResponseInitiated == -1) {
         timeToResponseInitiated = tempTime;
       }
-      if (skipped == -1 && timeToResponseLastRead == -1) {
+      // InputStream.skip will return 0 for both end of stream and for 0 bytes skipped.
+      boolean endOfStream = (skipped == 0 && byteCount != 0);
+      if (endOfStream && timeToResponseLastRead == -1) {
```

Comment:
This if statement will never be true.

I think it should be OK to move `networkMetricBuilder.setTimeToResponseCompletedMicros(timeToResponseLastRead);` into the previous if statement and remove this one.

Reply:
+1

Reply:
I am not sure that's right. Wouldn't it prematurely set `setTimeToResponseCompletedMicros` on the first seek. Then never set it again, even when seeking to the end of the stream?

Reply:
I'd misread the `timeToResponseLastRead`  to be the same as `timeToResponseInitiated`. This change LGTM.